api/envoy/type/matcher/v3/regex.proto

```diff
@@ -48,7 +48,7 @@ message RegexMatcher {
     //
     // .. note::
     //
-    //  The program size is still checked against the runtime value.
+    //  Although this field is deprecated, the program size will still be checked against the *max_program_size* runtime value.
```

Comment:
max_program_size is an inline literal and needs a double backtick - ie ` ``max_program_size `` `

Reply:
Oh Ok